.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: node_js
 
 node_js:
-  - '4.4.3'
+  - 'node'
```

Comment:
Shouldn't this suppose to be a node version?

Reply:
If you use `'node'` then it just uses the latest stable node version. You can check out how this works [here](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions).

I updated this because I saw that there was a `package-lock.json` file, but that would be ignored by node v4. I can change this to use a fixed version if you prefer.

Reply:
BTW it seems that there are some issues with nvm on travis, I think that is why this build failed.

You can see that the exact same build went fine for me [here](https://travis-ci.org/aberonni/wdio-selenium-standalone-service/builds/325525077) so maybe you just need to retrigger this build to get it fixed.